README.md

```diff
@@ -204,6 +204,24 @@ You can add or remove input and output variables at the stack level by following
 1.  Select a member configuration.
 1.  From the deployed details window, you can promote any of the configuration inputs or outputs.
 
+### Selective provisioning of observability resources
+
+You can selectively provision observability resources such as Activity Tracker routes and targets, and Cloud Monitoring instances by following these steps:
+
+1.  In the {{site.data.keyword.cloud_notm}} console, click the **Navigation menu** icon ![Navigation menu icon](/images/icon_hamburger.svg "Menu") > **Projects**.
```

Comment:
This isnt cloud docs, its markdown, so what you have in the curly bracket is not rendering to anything:
![image](https://github.com/user-attachments/assets/d4756c06-4520-42e5-8e4f-d997c5635e02)